src/components/cards/ListViewTaskCard.tsx

```diff
@@ -21,21 +40,59 @@ export const ListViewTaskCard = ({ task }: { task: TaskResponse }) => {
     >
       <AppMargin size={SizeofAppMargin.LARGE} py="6px">
         <Stack direction="row" columnGap={8} alignItems="center" justifyContent="space-between">
-          <Stack sx={{ width: '100%', cursor: 'pointer' }} direction="row" alignItems="center" columnGap={4}>
+          <Stack
+            sx={{ width: '100%', cursor: 'pointer' }}
+            direction="row"
+            alignItems="center"
+            columnGap={4}
+            onClick={handleClick}
+          >
             <Typography variant="bodyXs">WEB-01</Typography>
             <Typography variant="bodySm">{task?.title}</Typography>
           </Stack>
-          <Stack direction="row" alignItems="center" columnGap="20px" minWidth="fit-content">
+          <Stack direction="row" alignItems="center" columnGap="20px" minWidth="200px">
             <Box minWidth="fit-content">
               <Typography variant="bodySm">Apr 05, 2024</Typography>
             </Box>
-            <Stack direction="row" alignItems="center">
-              <Avatar
-                src={currentAssignee?.iconImageUrl || currentAssignee?.avatarImageUrl}
-                sx={{ width: '20px', height: '20px' }}
+            <Box minWidth="fit-content">
+              <Selector
+                getSelectedValue={(_newValue) => {
+                  const newValue = _newValue as IAssigneeCombined
+                  updateAssigneeValue(newValue)
+                  const assigneeType = newValue?.type
+                  if (updateTask) {
+                    updateTask({
+                      payload: {
+                        assigneeType:
+                          assigneeType === 'ius'
+                            ? 'internalUser'
+                            : assigneeType === 'clients'
+                              ? 'client'
+                              : assigneeType === 'companies'
+                                ? 'company'
+                                : 'internalUser',
```

Comment:
Use an object to lookup the value for this instead of nesting multiple ternary operators

```ts
const assigneeTypes = {
  ius: 'internalUser',
  clients: 'client',
  companies: 'company'
}
```
```ts
payload: { assigneeType: assigneeTypes[assigneeType] || 'internalUser' }
```